h2o-algos/src/test/java/hex/tree/gbm/GBMTest.java

```diff
@@ -661,6 +661,7 @@ private void checkUpdateAuxTreeWeights(GBMModel gbm, Frame frame) {
     Map<Integer, SharedTreeMojoModel.AuxInfo>[][] originalAuxInfos = visitTrees(
             gbm._output._treeKeysAux, Map.class, CompressedTree::toAuxInfos);
 
+    frame = Scope.track(ensureDistributed(frame, 16));
```

Comment:
Why 16 is the correct constant here? I am just curious...

Reply:
It guarantees that one node will have at least 3 chunks and it is a good number for this dataset (the chunks won't be empty).